Software-Engineering/L3-Senior-Software-Engineer.md

```diff
@@ -51,6 +51,7 @@
 - I manage difficult conversations and tactfully challenge others.
 - I improve the productivity and delivery of my team.
 - I help raise the bar for working at Octopus by providing regular code reviews and / or interviews for engineering candidates.
+- I facilitate healthy conflict through negotiation to arrive at positive, constructive outcomes.
```

Comment:
```suggestion
- I practice healthy conflict, using negotiation to arrive at positive, constructive outcomes.
```

💭 "Facilitate" can suggest a third-party or mediator role. At L3, I believe the intent is to emphasize that this person directly and constructively engages in conflicts, rather than just overseeing them. Consider using "practice" to reflect active participation.

Reply:
Great suggestion, it's a good call out that we want to lean into engagement vs overseeing, as facilitating those start to fall into leadership roles. Thanks for the suggestions I'll add them in 🙇